homeassistant/components/tuya/alarm_control_panel.py

```diff
@@ -149,7 +149,9 @@ def alarm_state(self) -> AlarmControlPanelState | None:
             self._master_state is not None
             and self.device.status.get(self._master_state.dpcode) == State.ALARM
         ):
-            return AlarmControlPanelState.TRIGGERED
+            # Only report as triggered if NOT a battery warning
+            if (changed_by := self.changed_by) is None or "Sensor Low Battery" not in changed_by:
```

Comment:
Ruff doesn't like the formatting
```suggestion
            if (
                changed_by := self.changed_by
            ) is None or "Sensor Low Battery" not in changed_by:
```

Reply:
Yes, already fixed.